src/transformers/image_utils.py

```diff
@@ -131,6 +131,13 @@ def convert_rgb(self, image):
 
         return image.convert("RGB")
 
+    def rescale(self, image: np.ndarray, scale: Union[float, int]) -> np.ndarray:
```

Comment:
The PR looks good overall but is undoing some of the changes in this [PR](https://github.com/huggingface/transformers/pull/18677), which renames the scale method to avoid clash with properties.

Could you update this file?